src/mixins/filing-template-mixin.ts

```diff
@@ -493,6 +493,11 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         }
       }) || []
     )
+    // Store business contact info
+    this.setBusinessContact({
+      ...entitySnapshot.authInfo.contact
```

Comment:
Why destructure into another object? To create a copy? Probably not needed as I assume the store creates its own copy.

(I see this is also done on line 298. Probably not needed there either.)

(There are definitely times when we want to create a copy/clone of an object, otherwise we can end up pointing to the same object as someone else, which can cause some difficult bugs to find.)

Reply:
Also lines 592-594 and 669-671 ?

Reply:
Good point! 
I think this was spread previously because we we applying `confirmEmail` at the same time. Since this is happening at the component level and we don't care about the confirmEmail beyond components validations we could probably just set it! 
Will try this, thank you

Reply:
Updated!